src/server.js

```diff
@@ -0,0 +1,40 @@
+const express = require("express");
+
+const logger = require("./logger");
+const { Resolver } = require("./resolver");
+
+class Server {
+  constructor() {
+    this.app = express();
+    this.resolver = new Resolver();
+    this.configure();
+  }
+
+  configure() {
+    this.app.get("/dnslink/:name", async (req, res) => {
+      const domainName = req.params.name;
+
+      try {
+        this.resolver.validateRequest(req);
+
+        const { skylink, sponsor } = await this.resolver.resolve(domainName);
+
+        res.json({ skylink, sponsor });
```

Comment:
Here is a good example of why documentation is important. Even after reading through the code it is not clear to me what the `sponsor` value is since the `Resolver` doesn't have documentation about what the return arguments are.

Reply:
I added JSdocs in [406df08](https://github.com/SkynetLabs/webportal-dnslink-api/pull/5/commits/406df08abb0e4fd10329083d1722779c6a7623eb) :)